tox.ini

```diff
@@ -1,5 +1,6 @@
 [tox]
-envlist = lint,py27,py34,doc,sphinx
+envlist = lint,py27,py34,py35,pep8py2,pep8py3,doc,sphinx
```

Comment:
is thre any difference between pep8py2 and pep8py3, and how do they differ from just pep8 ?